*** ./smali/com/android/internal/telephony/cdma/CDMAPhone.smali	2014-08-15 13:38:24.312255963 +0800
--- /home/art-chen/miuiv4/s7568/temp/new_smali/framework.jar.out/./smali/com/android/internal/telephony/cdma/CDMAPhone.smali	2014-08-15 13:40:26.928259247 +0800
***************
*** 1006,1011 ****
--- 1006,1015 ----
      .end annotation
  
      .prologue
+     const/16 v0, 0xf
+ 
+     invoke-virtual {p0, v0}, Lcom/android/internal/telephony/cdma/CDMAPhone;->removeMessages(I)V
+ 
      iget-object v0, p0, Lcom/android/internal/telephony/cdma/CDMAPhone;->mCT:Lcom/android/internal/telephony/cdma/CdmaCallTracker;
  
      invoke-virtual {v0}, Lcom/android/internal/telephony/cdma/CdmaCallTracker;->acceptCall()V
