*** ./smali/com/android/internal/policy/impl/PhoneFallbackEventHandler.smali	2014-08-15 13:38:32.516256183 +0800
--- /home/art-chen/miuiv4/s7568/temp/new_smali/android.policy.jar.out/./smali/com/android/internal/policy/impl/PhoneFallbackEventHandler.smali	2014-08-15 13:40:35.152259467 +0800
***************
*** 619,624 ****
--- 619,626 ----
  
      if-nez v4, :cond_2
  
+     invoke-virtual {p0}, Lcom/android/internal/policy/impl/PhoneFallbackEventHandler;->handleCameraKeyEvent()V
+ 
      goto :goto_0
  
      :sswitch_3
***************
*** 648,654 ****
  
      goto :goto_0
  
-     nop
  
      :sswitch_data_0
      .sparse-switch
--- 650,655 ----
***************
*** 753,755 ****
--- 754,766 ----
  
      goto :goto_0
  .end method
+ 
+ .method protected handleCameraKeyEvent()V
+     .locals 0
+     .annotation build Landroid/annotation/MiuiHook;
+         value = .enum Landroid/annotation/MiuiHook$MiuiHookType;->NEW_METHOD:Landroid/annotation/MiuiHook$MiuiHookType;
+     .end annotation
+ 
+     .prologue
+     return-void
+ .end method
\ 文件尾没有 newline 字符
