*** ./smali/android/preference/RingtonePreference.smali	2014-08-15 13:37:25.220254381 +0800
--- /home/art-chen/miuiv4/s7568/temp/new_smali/framework.jar.out/./smali/android/preference/RingtonePreference.smali	2014-08-15 13:39:24.808257583 +0800
***************
*** 311,316 ****
--- 311,322 ----
  
      invoke-virtual {p1, v0, v1}, Landroid/content/Intent;->putExtra(Ljava/lang/String;Ljava/lang/CharSequence;)Landroid/content/Intent;
  
+     const-string v0, "com.android.thememanager"
+ 
+     const-string v1, "com.android.thememanager.ThemeResourceTabActivity"
+ 
+     invoke-virtual {p1, v0, v1}, Landroid/content/Intent;->setClassName(Ljava/lang/String;Ljava/lang/String;)Landroid/content/Intent;
+ 
      return-void
  .end method
  
