*** ./smali/android/widget/TimePicker.smali	2014-08-15 13:37:03.276253793 +0800
--- /home/art-chen/miuiv4/s7568/temp/new_smali/framework.jar.out/./smali/android/widget/TimePicker.smali	2014-08-15 13:39:02.284256980 +0800
***************
*** 436,442 ****
      .parameter "x0"
  
      .prologue
!     invoke-direct {p0}, Landroid/widget/TimePicker;->onTimeChanged()V
  
      return-void
  .end method
--- 436,442 ----
      .parameter "x0"
  
      .prologue
!     invoke-virtual {p0}, Landroid/widget/TimePicker;->onTimeChanged()V
  
      return-void
  .end method
***************
*** 461,467 ****
      return-object v0
  .end method
  
! .method private onTimeChanged()V
      .locals 3
  
      .prologue
--- 461,467 ----
      return-object v0
  .end method
  
! .method onTimeChanged()V
      .locals 3
  
      .prologue
***************
*** 1187,1193 ****
  
      invoke-virtual {v0, v1}, Landroid/widget/NumberPicker;->setValue(I)V
  
!     invoke-direct {p0}, Landroid/widget/TimePicker;->onTimeChanged()V
  
      goto :goto_0
  
--- 1187,1193 ----
  
      invoke-virtual {v0, v1}, Landroid/widget/NumberPicker;->setValue(I)V
  
!     invoke-virtual {p0}, Landroid/widget/TimePicker;->onTimeChanged()V
  
      goto :goto_0
  
***************
*** 1232,1238 ****
  
      invoke-virtual {v0, v1}, Landroid/widget/NumberPicker;->setValue(I)V
  
!     invoke-direct {p0}, Landroid/widget/TimePicker;->onTimeChanged()V
  
      goto :goto_0
  .end method
--- 1232,1238 ----
  
      invoke-virtual {v0, v1}, Landroid/widget/NumberPicker;->setValue(I)V
  
!     invoke-virtual {p0}, Landroid/widget/TimePicker;->onTimeChanged()V
  
      goto :goto_0
  .end method
