*** ./smali/android/widget/TimePicker$4.smali	2014-08-15 13:36:55.228253578 +0800
--- /home/art-chen/miuiv4/s7568/temp/new_smali/framework.jar.out/./smali/android/widget/TimePicker$4.smali	2014-08-15 13:38:53.556256746 +0800
***************
*** 65,70 ****
--- 65,74 ----
      #calls: Landroid/widget/TimePicker;->updateAmPmControl()V
      invoke-static {v0}, Landroid/widget/TimePicker;->access$200(Landroid/widget/TimePicker;)V
  
+     iget-object v0, p0, Landroid/widget/TimePicker$4;->this$0:Landroid/widget/TimePicker;
+ 
+     invoke-virtual {v0}, Landroid/widget/TimePicker;->onTimeChanged()V
+ 
      return-void
  
      :cond_0
