*** ./smali/com/android/internal/view/menu/ActionMenuItemView.smali	2014-08-15 13:38:12.356255643 +0800
--- /home/art-chen/miuiv4/s7568/temp/new_smali/framework.jar.out/./smali/com/android/internal/view/menu/ActionMenuItemView.smali	2014-08-15 13:40:14.784258921 +0800
***************
*** 133,138 ****
--- 133,146 ----
  
      iget-object v3, p0, Lcom/android/internal/view/menu/ActionMenuItemView;->mItemData:Lcom/android/internal/view/menu/MenuItemImpl;
  
+     invoke-virtual {v3}, Lcom/android/internal/view/menu/MenuItemImpl;->isForceShowText()Z
+ 
+     move-result v3
+ 
+     if-nez v3, :cond_0
+ 
+     iget-object v3, p0, Lcom/android/internal/view/menu/ActionMenuItemView;->mItemData:Lcom/android/internal/view/menu/MenuItemImpl;
+ 
      invoke-virtual {v3}, Lcom/android/internal/view/menu/MenuItemImpl;->showsTextAsAction()Z
  
      move-result v3
