*** ./smali/android/content/res/Resources$Theme.smali	2014-08-15 13:37:26.768254422 +0800
--- /home/art-chen/miuiv4/s7568/temp/new_smali/framework.jar.out/./smali/android/content/res/Resources$Theme.smali	2014-08-15 13:39:26.372257625 +0800
***************
*** 1,4 ****
! .class public final Landroid/content/res/Resources$Theme;
  .super Ljava/lang/Object;
  .source "Resources.java"
  
--- 1,4 ----
! .class public Landroid/content/res/Resources$Theme;
  .super Ljava/lang/Object;
  .source "Resources.java"
  
***************
*** 9,15 ****
  .end annotation
  
  .annotation system Ldalvik/annotation/InnerClass;
!     accessFlags = 0x11
      name = "Theme"
  .end annotation
  
--- 9,15 ----
  .end annotation
  
  .annotation system Ldalvik/annotation/InnerClass;
!     accessFlags = 0x1
      name = "Theme"
  .end annotation
  
